src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -124,16 +124,32 @@
 
 		<section class="statistics section text-center">
 			<div class="row">
-				<h2 class="statistics__header small-12 columns">
-					Statistics
-				</h2>
+				<div class="small-12 columns">
+					<kv-responsive-image
+						class="statistics__img"
+						:images="statistics"
+						loading="lazy"
+						alt="statistics image"
```

Comment:
Since this image is just for decoration (it doesn't add meaning for a user), and empty alt tag, `alt=""`, will keep the image from announcing to screenreaders.